File: Hotelix.Mobile/Models/Hotel.cs
namespace Hotelix.Mobile.Models;

public class Hotel
{
	public int Id { get; set; }
	public string? Name { get; set; }
	public string? Description { get; set; }
	public decimal PricePerNight { get; set; }
	public bool HasInternet { get; set; }
	public bool HasTelevision { get; set; }
	public bool HasParking { get; set; }
	public bool HasCafeteria { get; set; }
	public string? CoverImage { get; set; }
	public Address? Address { get; set; }
	public Contact? Contact { get; set; }
}

File: Hotelix.Web/src/app/models/hotel.ts
import { Address } from "./address";
import { Contact } from "./contact";

export interface Hotel {
    id: number;
    name: string;
    description: string;
    pricePerNight: number;
    hasInternet: boolean;
    hasTelevision: boolean;
    hasParking: boolean;
    hasCafeteria: boolean;
    coverImage: string;
    address: Address;
    contact: Contact;
}
